clientgui/DlgAdvPreferences.cpp

```diff
@@ -679,35 +682,60 @@ bool CDlgAdvPreferences::ValidateInput() {
         ShowErrorMessage(invMsgLimit100, m_txtProcUseProcessors);
         return false;
     }
-    
+
     buffer = m_txtProcUseCPUTime->GetValue();
     if(!IsValidFloatValueBetween(buffer, 0.0, 100.0)) {
         ShowErrorMessage(invMsgLimit100, m_txtProcUseCPUTime);
         return false;
     }
-    
-    if(m_txtProcIdleFor->IsEnabled()) {
-        buffer = m_txtProcIdleFor->GetValue();
-        if(!IsValidFloatValueBetween(buffer, 0, 10000)) {
-            ShowErrorMessage(invMsgFloat,m_txtProcIdleFor);
-            return false;
-        }
-    }
 
-    buffer = m_txtNoRecentInput->GetValue();
-    if (!IsValidFloatValueBetween(buffer, 0, 10000)) {
+    // Validate No Recent Input
+    //
+    bufferNRI = m_txtNoRecentInput->GetValue();
+    if (!IsValidFloatValueBetween(bufferNRI, 0, 10000)) {
```

Comment:
I would recommend to use named constant/define/constexpr instead of magic numbers. For example, 100.0 in ```IsValidFloatValueBetween(buffer, 0.0, 100.0)``` is intuitively understandable, but 10000 here is not.